config.rb

```diff
@@ -13,7 +13,7 @@
 page '/*.json', layout: false
 page '/*.txt', layout: false
 
-config[:http_prefix] = '/intranet'
+config[:http_prefix] = ''
```

Comment:
When I tried it without this, it no longer loaded the css file in production, as it was being requested from `citizensadvice.github.io/assets/stylesheets...`